docker/cassandra-operator/Dockerfile.template

```diff
@@ -1,6 +1,5 @@
-ARG openjre_image
-
-FROM ${openjre_image}
+# please execute 'make' in this directory to build this image
+FROM {openjre_image}
```

Comment:
I cant build this image, maybe I have just different  docker  version but according to spec / docs FROM should be the very first command otherwise it  fails

Reply:
This is supported by newer Docker versions: https://docs.docker.com/engine/reference/builder/#understand-how-arg-and-from-interact